docker-compose/monitor/README.md

```diff
@@ -0,0 +1,183 @@
+# Aggregated Trace Metrics Development Environment
+
+Development and demo environment for Aggregated Trace Metrics.
+
+This environment consists four backend components:
+
+- [MicroSim](https://github.com/yurishkuro/microsim): a program to simulate traces.
+- [Jaeger All-in-one](https://www.jaegertracing.io/docs/1.24/getting-started/#all-in-one): the full Jaeger stack in a single docker image.
+- [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/): vendor agnostic integration layer for traces and metrics. Its main role in this particular development environment is to receive Jaeger spans, forward these spans untouched to Jaeger All-in-one while simultaneously aggregating metrics out of this span data. To learn more about span metrics aggregation, please refer to the [spanmetrics processor documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor).
+- [Prometheus](https://prometheus.io/): a metrics collection and query engine, used to scrape metrics computed by OpenTelemetry Collector, and presents an API for Jaeger All-in-one to query these metrics.
+
+![ATMDev (1)](https://user-images.githubusercontent.com/26584478/127763924-4ae2ce88-8fc2-4def-90c2-55358a433905.png)
+
+# Getting Started
+
+## Bring up/down the dev environment
+```
+docker compose up
+docker compose down
+```
+
+## Example 1
+Fetch call rates for both the driver and frontend services, grouped by operation, from now,
+looking back 1 second with a sliding rate-calculation window of 1m and step size of 1 millisecond
+
+```bash
+curl http://localhost:16686/api/metrics/calls\?service\=driver\&service\=frontend\&groupByOperation\=true\&endTs\="$(date +%s)"000\&lookback\=1000\&step\=100\&ratePer\=60000 | jq .
+```
+
+
+## Example 2
+Fetch P95 latencies for both the driver and frontend services from now,
+looking back 1 second with a sliding rate-calculation window of 1m and step size of 1 millisecond, where the span kind is either "server" or "client".
+
+```bash
+curl http://localhost:16686/api/metrics/latencies\?service\=driver\&service\=frontend\&quantile\=0.95\&endTs\="$(date +%s)"000\&lookback\=1000\&step\=100\&ratePer\=60000\&spanKind\=server\&spanKind\=client | jq .
+```
+
+## Example 3
+Fetch error rates for both driver and frontend services using default parameters.
+```bash
+curl http://localhost:16686/api/metrics/errors\?service\=driver\&service\=frontend | jq .
+```
+
+## Example 4
+Fetch the minimum step size supported by the underlying metrics store.
+```bash
+curl http://localhost:16686/api/metrics/minstep | jq .
+```
+
+# HTTP API
```

Comment:
I'm planning to add this section into https://www.jaegertracing.io/docs/latest/apis/, would that be an appropriate place or is it sufficient to leave this API doc here?